samples/qa-a11y/src/App/App.js

```diff
@@ -46,45 +67,101 @@ const Menu = SpotlightContainerDecorator({enterTo: 'last-focused'}, 'div');
 const views = [
 	{title: 'About qa-a11y', view: Home},
 	{debugProps: true, title: 'Option', view: Option},
+	{title: 'ArcPicker', view: ArcPicker},
+	{title: 'ArcSlider', view: ArcSlider},
 	{title: 'Button', view: Button},
+	{title: 'BodyText', view: BodyText},
 	{title: 'CheckboxItem', view: CheckboxItem},
 	{title: 'ColorPicker', view: ColorPicker},
 	{title: 'ContextualPopupDecorator', view: ContextualPopupDecorator},
+	{title: 'DatePicker', view: DatePicker},
 	{title: 'DateTimePicker', view: DateTimePicker},
 	{title: 'Drawer', view: Drawer},
 	{title: 'Dropdown', view: Dropdown},
+	{title: 'FanSpeedControl', view: FanSpeedControl},
+	{title: 'FullscreenPopup', view: FullscreenPopup},
 	{isHeader: false, title: 'Header', view: Header},
+	{title: 'Heading', view: Heading},
 	{title: 'ImageItem', view: ImageItem},
+	{title: 'IncrementSlider', view: IncrementSlider},
 	{title: 'Input', view: Input},
 	{title: 'Item', view: ItemView},
+	{title: 'Keypad', view: Keypad},
+	{title: 'LabeledIconButton', view: LabeledIconButton},
+	{title: 'MediaPlayer', view: MediaPlayer},
 	{isHeader: false, title: 'Panels', view: Panels},
 	{title: 'Picker', view: Picker},
 	{title: 'Popup', view: Popup},
+	{title: 'PopupMenu', view: PopupMenu},
 	{title: 'ProgressBar', view: ProgressBar},
 	{title: 'RadioItem', view: RadioItem},
 	{title: 'RangePicker', view: RangePicker},
 	{title: 'ReadAlert', view: ReadAlert},
 	{isHeader: false, title: 'Scroller', view: Scroller},
 	{title: 'Slider', view: Slider},
+	{title: 'SliderButton', view: SliderButton},
 	{title: 'Spinner', view: Spinner},
 	{title: 'SwitchItem', view: SwitchItem},
+	{isHeader: false, title: 'TabbedPanels', view: TabbedPanelsView},
+	{title: 'TabGroup', view: TabGroup},
+	{title: 'TemperatureControl', view: TemperatureControl},
+	{title: 'ThumbnailItem', view: ThumbnailItem},
+	{title: 'TimePicker', view: TimePicker},
+	{title: 'ToggleButton', view: ToggleButton},
 	{title: 'TooltipDecorator', view: TooltipDecorator},
 	{isHeader: false, title: 'VirtualGridList', view: VirtualGridList},
-	{isHeader: false, title: 'VirtualList', view: VirtualList}
+	{isHeader: false, title: 'VirtualList', view: VirtualList},
+	{title: 'WindDirectionControl', view: WindDirectionControl}
+];
+
+const tabTitles = [
+	{title: 'A-G'},
+	{title: 'H-N'},
+	{title: 'O-S'},
+	{title: 'T-Z'}
 ];
 
 class AppBase extends React.Component {
 	static propTypes = {
 		rtl: PropTypes.bool,
 		updateLocale: PropTypes.func
 	};
+
 	constructor () {
 		super();
+
 		this.state = {
 			isDebugMode: false,
 			jumpToView: '',
 			selected: 0
 		};
+
+		this.viewItems = [];
+		this.viewItems.length = tabTitles.length;
+
+		views.forEach((view, indexInViews) => {
+			const item = (
+				<Item
+					aria-label={view.title}
+					className={appCss.navItem}
+					data-menu={indexInViews}
+					key={indexInViews}
+					onClick={this.handleChangeView(indexInViews)}
+					slotBefore={('00' + indexInViews).slice(-2)}
```

Comment:
Could we have space between the number and the title?

Reply:
Applied wider width for the slotBefore.